### DISCUSSION TITLE
[EASY] Bypass Google security

### DISCUSSION THREAD

**Discussion by HugoMskn** [qnblackcat/uYouPlus#922] [General]
Hello folks

As some of you might have experienced you can't login anymore 

But let me tell you a way to do it wayyy easier 

Let's get started 

So first you need :
A pc with iTunes and sideloadly etc.. Latest ipa 
Then make sure the account exist on the phone before installing the IPA

Check theses requirements before starting, remove vanilla and cracked YouTube as well

[See picture below to understand]
![CF83C3BD-19E4-43AE-B57F-3411611DE675](https://user-images.githubusercontent.com/81116453/216468986-6de82e5f-a23e-48a5-874d-0197f6602863.png)

Once requirements are good, open sideloadly, select the IPA, and then click advanced options (1), then look down to" inject dylibs", (2) and then click Sideload spoofer (3) then start, wait until it's done then you will just need to select the account

Then wooo Google security bypassed ! You can now use YouTube freely! 

If you have any question reach me on discord : Hugo ;)#3737 or telegram : https://t.me/HugoPML
![D45828D7-6E96-482C-A5B6-2617ED4ED335](https://user-images.githubusercontent.com/81116453/216469008-62fbafe4-2781-4c8e-a0a9-681cc68f1a12.png)

**Comment by yodaluca23** [comment]
@qnblackcat Could you do this with new releases? Because this would probably make a lot of users more confident about uYou+...

**Comment by swayea** [comment]
Confirmed, works on iPhone 11 iOS 16.3 

Update : Works on 16.1.2 too

**Comment by Ephtar** [comment]
Thanks a lot ! Working ! Ipad 6th gen, 14.8.1
